src/System.Security.Cryptography.Encoding/src/Configurations.props

```diff
@@ -3,8 +3,9 @@
   <PropertyGroup>
     <BuildConfigurations>
       net463-Windows_NT;
+      netstandard-OSX;
```

Comment:
Yep, the Unix one (which includes FreeBSD/NetBSD) depends on OpenSSL, the OSX one doesn't.